src/editor/src/UI.cpp

```diff
@@ -6,6 +6,7 @@
 #include <imgui/imgui_internal.h>
 #include "ContentPanel.hpp"
 #include "DebugUI.hpp"
+#include "DebugTooltip.hpp"
 
 std::vector<std::unique_ptr<Hush::IEditorPanel>> Hush::UI::S_ACTIVE_PANELS{};
```

Comment:
Same question, do we need a static variable?

I think that we should aim for local variables, especially as this might be part of the UI class.

As for the UI class, I saw that it also has a bunch of static members, we could create an instance of that class and make all of their functions part of the class instance. The part of initialize panels would be part of the constructor, so we avoid having to manually initialize it (which is prone to errors)